CRM/CiviAwards/Helper/CaseTypeCategory.php

```diff
@@ -55,13 +55,27 @@ public static function getApplicantManagementCaseCategories() {
   }
 
   /**
-   * Returns the option values for Award Subtypes.
+   * Returns the option values for Award Subtypes that has been used.
    *
    * @return array
    *   Award Subtypes.
    */
   public static function getSubTypes() {
-    return AwardDetail::buildOptions('award_subtype');
+    $usedSubTypes = civicrm_api3('AwardDetail', 'get', [
+      'return' => ['award_subtype'],
+      'options' => ['limit' => 0],
+    ]);
+    if (empty($usedSubTypes['values'])) {
+      return [];
+    }
+    $usedSubTypes = array_column($usedSubTypes['values'], 'award_subtype');
```

Comment:
This list could be pretty long and be full of duplicates for the award_subtypes values, we can reduce the duplicates at this point. 
```php
$usedSubTypes = array_unique(array_column($usedSubTypes['values'], 'award_subtype'));
```

Reply:
changed